src/graphics/webgl/webgl-graphics-device.js

```diff
@@ -568,6 +568,9 @@ class WebglGraphicsDevice extends GraphicsDevice {
         }
 
         this.constantTexSource = this.scope.resolve("source");
+        this.textureBias = this.scope.resolve("textureBias");
+
+        this.textureBias.setValue(0.0);
```

Comment:
I think we should have a public API for this. Ideally the viewer code you test it on would not do
`this.scope.resolve("textureBias").SetValue(...);`

Reply:
This transform feedback example already uses this API. This uniform management is _long_ overdue a redesign.

Reply:
That's true, but transform feedback example is accessing this API to set uniforms in its own shader.
But here we're adding new functionality to the engine that the Viewer wants to use .. and that should be different API, allowing us to even rename the variable later if needed.  Maybe `Scene.standardMaterialTextureBias' or a shorter version of it? Something we publicly document.

Reply:
I'd prefer something like `Scene#textureBias` and then potentially add `Texture#bias` which would take precedence.

Reply:
Could be a scene setting... though I would prefer creating and exposing public API in followup PR.

Reply:
Fine by me.